release/device-rfid-llrp-go.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: 'device-rfid-llrp-go'
+version: '1.0.0'
+releaseName: 'ireland'
```

Comment:
Should be `hanoi` since it is a 1.x release

Reply:
Not sure we use `releaseName` for anything actually but I changed it to be consistent.